specs/architecture.html

```diff
@@ -348,7 +348,7 @@
       	  <tr>
       	    <td>Payment App</td>
       	    <td>Digital Wallet, Payment Instrument </td>
-      	    <td>A piece of software to pay</td>
+      	    <td>A piece of software to manage payment methods</td>
```

Comment:
It's more than that really, no? It's either: a piece of software that does pay or returns back a payment credential to enable payment.